gn/BUILDCONFIG.gn

```diff
@@ -45,6 +49,7 @@ is_linux = current_os == "linux"
 is_mac = current_os == "mac"
 is_win = current_os == "win" || current_os == "winrt"
 is_winrt = current_os == "winrt"
+is_tizen = building_for_tizen
```

Comment:
Maybe do `is_tizen = current_os == "tizen"`. I am assuming that you are using the Linux target, so you can update that to `is_linux = current_os == "linux" || current_os == "tizen"`. There no real reason other than consistency at this point.

Reply:
Done.